src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -96,7 +96,7 @@ public function panelAction(Request $request, $token)
         }
 
         if (!$profile = $this->profiler->loadProfile($token)) {
-            return new Response($this->twig->render('@WebProfiler/Profiler/info.html.twig', array('about' => 'no_token', 'token' => $token)), 200, array('Content-Type' => 'text/html'));
+            return new Response($this->twig->render('@WebProfiler/Profiler/info.html.twig', array('about' => 'no_token', 'token' => $token, 'request' => $request)), 200, array('Content-Type' => 'text/html'));
```

Comment:
Shouldn't we change this to a redirect to `_profiler_info`, just like is done with the other info messages?